app/javascript/kramdown/lexer.test.js

```diff
@@ -0,0 +1,183 @@
+        { type: "close_tag", value: "test", raw: "{:/test}" },
+      ],
+    },
+    {
+      input: 'foo {::test attr="value \\"1\\""}bar{:/test} baz',
+      expected: [
+        { type: "text", value: "foo " },
+        { type: "start_open_tag", value: "test", raw: "{::test " },
+        { type: "attr_name", value: "attr", raw: "attr=" },
+        { type: "attr_value", value: 'value "1"', raw: '"value \\"1\\""' },
+        { type: "end_open_tag", raw: "}" },
+        { type: "text", value: "bar" },
+        { type: "close_tag", value: "test", raw: "{:/test}" },
+        { type: "text", value: " baz" },
+      ],
+    },
+    {
+      input: "{::outer}{::inner}{:/outer}{:/inner}",
+      expected: [
+        { type: "start_open_tag", value: "outer", raw: "{::outer" },
+        { type: "end_open_tag", raw: "}" },
+        { type: "start_open_tag", value: "inner", raw: "{::inner" },
+        { type: "end_open_tag", raw: "}" },
+        { type: "close_tag", value: "outer", raw: "{:/outer}" },
+        { type: "close_tag", value: "inner", raw: "{:/inner}" },
+      ],
+    },
+    {
+      input: '{::test a="foo"}',
+      expected: [
+        { type: "start_open_tag", value: "test", raw: "{::test " },
+        { type: "attr_name", value: "a", raw: "a=" },
+        { type: "attr_value", value: "foo", raw: '"foo"' },
+        { type: "end_open_tag", raw: "}" },
+      ],
+    },
+
+    {
+      input: '{::test a.="foo"}',
+      expected: [
+        { type: "start_open_tag", value: "test", raw: "{::test " },
+        { type: "text", value: 'a.="foo"}' },
+      ],
+    },
+    {
+      input: '{::test attr.="foo"}',
+      expected: [
+        { type: "start_open_tag", value: "test", raw: "{::test " },
+        { type: "text", value: 'attr.="foo"}' },
+      ],
+    },
+    {
+      input: "{::test}foo{:/?}",
+      expected: [
+        { type: "start_open_tag", value: "test", raw: "{::test" },
+        { type: "end_open_tag", raw: "}" },
+        { type: "text", value: "foo{:/?}" },
+      ],
+    },
+    {
+      input: "{::test}foo{:/a?}",
+      expected: [
+        { type: "start_open_tag", value: "test", raw: "{::test" },
+        { type: "end_open_tag", raw: "}" },
+        { type: "text", value: "foo" },
+        { type: "text", value: "{:/a?}" },
+      ],
+    },
+    {
+      input: "{::test /FOO",
+      expected: [{ type: "text", value: "{::test /FOO" }],
+    },
+    {
+      input: "{::test attr=foo}",
+      expected: [
+        { type: "start_open_tag", value: "test", raw: "{::test " },
+        { type: "text", value: "attr=foo}" },
+      ],
+    },
+    {
+      input: "{::foo ?",
+      expected: [{ type: "text", value: "{::foo ?" }],
+    },
+    {
+      input: "{::foo?",
+      expected: [{ type: "text", value: "{::foo?" }],
+    },
+    {
+      input: "{::_",
+      expected: [{ type: "text", value: "{::_" }],
+    },
+  ];
+
+  tests.forEach(({ input, expected }) => {
+    // eslint-disable-next-line
```

Comment:
[question] does this need a rule name to disable? Or does this disable any and every rule on the next line?

Reply:
Without a rule name this will disabled the linter on the next line. I think this was just the linter complaining that `test` wasn't receiving a string literal as a test name. I can improve the scope on this

Reply:
Ah, it looks like the rule I was disabling was [`jest/valid-title`](https://github.com/jest-community/eslint-plugin-jest/blob/v26.1.5/src/rules/valid-title.ts), which just wants _some_ part of the test title to be a string literal. I am updating to meet those requirements.